doc/fish-completion.fish

```diff
@@ -79,7 +79,7 @@ function __restic_clear_perform_completion_once_result
     __restic_debug ""
     __restic_debug "========= clearing previously set __restic_perform_completion_once_result variable =========="
     set --erase __restic_perform_completion_once_result
-    __restic_debug "Succesfully erased the variable __restic_perform_completion_once_result"
+    __restic_debug "Successfully erased the variable __restic_perform_completion_once_result"
```

Comment:
This file is generated by restic as part of the release preparation process, so we shouldn't edit it directly :)

Regardless, the typo in question was from the Cobra package and has been fixed here: https://github.com/spf13/cobra/commit/284f4101043c4b1faebde411caec35a7b6e36494

So this change isn't something we do in restic, can you remove this change please?

Reply:
removed